mkdocs.yml

```diff
@@ -37,7 +37,7 @@ pages:
 - Scanning for Extension Points: scanning-for-extension-points.md
 - Bootstrapper: bootstrapper.md
 - Advanced usage: advanced-usage.md
-- Analyzer: analyzer.md
+- Analyzer: https://fakeiteasyanalyzers.readthedocs.io/

```

Comment:
1. I figured there's no point in directing people to our obsolete analyzers.md page
2. Based on the subproject directory structure, I saw no way to make a decent relative link to the analyzer docs, so I used a raw URL like anybody else would

Reply:
I didn't know there was a top-level URL for subprojects. I only knew of this one: https://fakeiteasy.readthedocs.io/projects/fakeiteasyanalyzers/en/latest/.

Maybe you could link `/projects/fakeiteasyanalyzers/` instead? this way it would still appear "under" the FakeItEasy root doc site. And maybe we could change the alias to just "analyzers".

Reply:
subprojects are projects in their own right, so there's a top-level.
I didn't even pick the original subproject URL, and when I tried to change, it got all snippy with me. Ended up deleting and readding after you asked for the change.